test/assets/release.json

```diff
@@ -33,6 +33,40 @@
   "created_at": "2020-02-20T12:44:25Z",
   "published_at": "2020-03-10T09:42:12Z",
   "assets": [
+    {
+      "url": "http://api.github.com/repos/clangd/clangd/releases/assets/18600444",
+      "id": 18600444,
+      "node_id": "MDEyOlJlbGVhc2VBc3NldDE4NjAwNDQ0",
+      "name": "clangd-indexing-tools-linux-10.zip",
```

Comment:
Did you mean `clangd_indexing_tools-linux-10.zip`?

Reply:
(sorry about the long delay here)
No, because if we use clangd_indexing_tools-linux-10.zip then the test passes without the code change :-) Because if there are multiple matches it picks the first.

IIRC, We had two different fixes for the same issue: renaming the file and this patch.
Is this too confusing?

Reply:
The confusing part is that we don't have the naming convention for the indexing tools like that anymore, I think. But it is probably OK.